### DISCUSSION TITLE
Phi2 ODF Section Labels

### DISCUSSION THREAD

**Discussion by bnmcbride** [mtex-toolbox/mtex#645] [Ask Anything]
Is there a way to relocate the Phi2 label on ODF section plots? I saw similar discussion abut changing font size or suppressing these labels (https://github.com/mtex-toolbox/mtex/issues/239) but I'm not sure who to access the Phi2 labels to reposition them. 

This issue I have is that the default location of the Phi2 labels often overlaps strong texture components and makes the plot a bit difficult to read. I'd like to be able to relocate the Phi2 label centered above the plot.

Currently I've just been using:

`plot(odf,'sections',9,'silent')
setColorRange('equal')
mtexColorbar('title','M.U.D.')`

![image](https://user-images.githubusercontent.com/73897823/102781541-121b6b00-4355-11eb-8e76-999b74d92e6c.png)

**Comment by kilir** [comment]
Hi,
how about:
```
plot(uniformODF)
f = gcm
for i=1:length(f.children)
 sp = getappdata(f.children(i),'sphericalPlot')
 sp.TR.Position(2) = 1.1 * sp.TR.Position(2);
 sp.TR.Position(1) = 0.95 * sp.TR.Position(1);
 sp.TR.FontSize=24;
 sp.TR.Color='r';
end
```
Here the positions are just shifted by a factor, but of course you could add a constant or whatever fits.

Cheers,
Rüdiger
